.github/workflows/arm64.yml

```diff
@@ -83,6 +86,14 @@ jobs:
             --ignore=tests/scenarios/test_sync.py \
             --ignore=tests/scenarios/test_transformation.py \
             -k "not test_search"
+      - run: echo "TEST_FAILURE=true" >> $GITHUB_ENV
+        if: failure()
+      - name: Create JIRA issue
+        if: always()
+        run: |
+          pip install -r ./scripts/ci/requirements.txt
+          ./scripts/ci/create_jira_issue.py
```

Comment:
Did we test this by running the ARM64 ACS Packaging CI via `gh workflow run .github/workflows/arm64.yml  --ref this/branch-name`? I believe this shouldn't work given that we're checking out an older tag for tests which won't contain these files (hence why the workarounds related to `dtas-config.json` and the pipeline-all-amps docker-compose can be seen in previous steps.

In case it doesn't work (as I'd expect), we could temporarily apply the same workaround for these files as well, saving these files before checking out the test tag and moving them in their appropriate location after the test tag has been checked out. Once a new -Ax tag of ACS is available we can remove the workaround as they will be there since now on.

Reply:
Note: alternatively we could checkout the `master` branch again before running the JIRA integration, via the `checkout` action.

Reply:
I used the `checkout` action.
Ref. https://github.com/Alfresco/acs-packaging/pull/2450